internal/core/data/event_test.go

```diff
@@ -66,12 +65,12 @@ func TestCountByDevice(t *testing.T) {
 	}
 }
 
-func buildEvents() []models.Event {
-	events := []models.Event{}
-	events = append(events, models.Event{
+func buildEvents() []contract.Event {
+	events := []contract.Event{}
+	events = append(events, contract.Event{
 		ID:     "1",
 		Device: testDeviceName,
-		Readings: []models.Reading{
+		Readings: []contract.Reading{
 			{Id: "1"},
 			{Id: "2"},
 		},
```

Comment:
Nitpick.  This could be reduced to:

```
func buildEvents() []contract.Event {
	return []contract.Event{
		{
			ID:     "1",
			Device: testDeviceName,
			Readings: []contract.Reading{
				{Id: "1"},
				{Id: "2"},
			},
		},
	}
}
```